src/stravalib/model.py

```diff
@@ -871,9 +875,8 @@ def full_photos(self) -> BatchedResultsIterator[ActivityPhoto]:
 class SummaryActivity(MetaActivity, strava_model.SummaryActivity):
     # field overrides from superclass for type extensions:
     athlete: Optional[MetaAthlete] = None
-    start_latlng: Optional[LatLon] = (
-        None  # Do we still need these overrides for latlon?
-    )
+    # These force validator to run on lat/lon
```

Comment:
i wasn't sure why there was this output with tuple like brackets so i removed it. again there could be some meaning here that i'm missing. we do need the overrides because of how the generated model gets typed. it does not allow None as typed.